### PR TITLE
NodeJS / Web3 load runner tool

### PR DESCRIPTION
**Pull request by illya13**
- [x] NodeJS / Web3 load runner tool
- [x] `docker-compose` based P2P network
- [x] Travis-CI integration

### CODE REVIEW

File: test/p2p/mach4/chaindata/LOG [hunk 1]
```diff
@@ -0,0 +1,27 @@
+=============== Feb 19, 2017 (EET) ===============
```
Comment (melekes) [inline test/p2p/mach4/chaindata/LOG]: do we need `chaindata` in the repo?

Comment (illya13) [reply test/p2p/mach4/chaindata/LOG]: maybe we don't :(
put here for simplicity, will try to avoid it

**Review by melekes — APPROVED** [review]

File: test/contract/deploy.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const config = require('config');
+const ABI = require('./config/ABI.json');
+const Web3 = require('web3');
+
+const _web3 = new Web3(new Web3.providers.HttpProvider(config.get('provider')));
+_web3._extend({
```
Comment (ebuchman) [inline test/contract/deploy.js:6]: what's this about ?

File: test/contract/deploy.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
...815260200160002060020160009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff167fadf710a8ef6202ad47d4f3fe8844c285e392dc2dbc2797dd4859dedbd1fd5e8360006000858152602001908152602001600020600001546000600086815260200190815260200160002060010160405180838152602001806020018281038252838181546001816001161561010002031660029004815260200191508054600181600116156101000203166002900480156106215780601f106105f657610100808354040283529160200191610621565b820191906000526020600020905b81548152906001019060200180831161060457829003601f168201915b5050935050505060405180910390a25b5050565b602060405190810160405280600081525090565b602060405190810160405280600081525090565b828054600181600116156101000203166002900490600052602060002090601f016020900481019282601f1061069e57805160ff19168380011785556106cc565b828001600101855582156106cc579182015b828111156106cb5782518255916020019190600101906106b0565b5b5090506106d99190610709565b5090565b815481835581811511610704578183600052602060002091820191016107039190610709565b5b505050565b61072b91905b8082111561072757600081600090555060010161070f565b5090565b905600a165627a7a72305820be1a201c1518079c9986fad0961fa10136252c5e377fbabfe6eb9ee1009e787d0029',
```
Comment (ebuchman) [inline test/contract/deploy.js:24]: wouldnt it be better to use solc in here?

File: test/contract/deploy.js (hunk 2 above)
Comment (illya13) [reply test/contract/deploy.js:6]: ```_web3._extend``` ?

File: test/contract/deploy.js (hunk 3 above)
Comment (illya13) [reply test/contract/deploy.js:24]: it might be

**Comment by adrianbrink** [comment]
@illya13

**Pull request closed without merging**